scripts/docker/build_docker.py

```diff
@@ -307,39 +529,66 @@ def __parse_arguments(args_list):
     docker_remote_args_group = parser.add_argument_group(
         'Docker push', 'controlling behavior related pushing dockers to remote repos'
     )
-    docker_remote_args_group.add_argument('--gcr-project', type=str,
-                                          help='GCR billing project to push the images to. If not given, the '
-                                               'built image(s) will not be push to GCR.')
+    docker_remote_args_group.add_argument('--docker-repo', type=str, help='Docker repo to push images to.')
     docker_remote_args_group.add_argument('--update-latest', action='store_true',
-                                          help='also update \"latest\" tag in remote docker repo(s)')
-    # flag to turn off git protection (default mode is refusing to build when there are untracked files and/or
-    # uncommitted changes)
+                                          help=f'also update \"{ProjectBuilder.latest_tag}\" tag in remote docker'
+                                               f'repo(s)')
+    dockers_json_path = os.path.join(Paths.gatk_sv_path, "input_values", "dockers.json")
+    docker_remote_args_group.add_argument('--input-json', type=str, default=dockers_json_path,
+                                          help="Path to dockers.json to use as input")
+    docker_remote_args_group.add_argument('--output-json', type=str, default=dockers_json_path,
```

Comment:
I don't agree with that characterization (it's not a flag). The current behavior of the pipeline does what I'm guessing we nearly always want (update dockers.json in place) with no arguments. With one argument you can tell it to update a different file, or not update. I don't think adding more arguments will make it easier to use or understand. If you insist I can change it to use the three-argument system you specify though.

Reply:
No strong preference, I guess your implementation and my suggestion are both valid design decisions, and since you're the one implementing this, your preference supersedes mine :)